testing/test_config.py

```diff
@@ -1976,7 +1976,8 @@ def test_config_blocked_default_plugins(pytester: Pytester, plugin: str) -> None
         assert result.ret == ExitCode.USAGE_ERROR
         result.stderr.fnmatch_lines(
             [
-                "ERROR: found no collectors for */test_config_blocked_default_plugins.py",
+                "ERROR: not found: */test_config_blocked_default_plugins.py",
+                "(no match in any of *<Dir *>*",
```

Comment:
this error seems a little worse, but thats good for a follow up

Reply:
The difference is because previously there were no collectors in the path to the `Module`, so a generic fallback message was shown, but now there is the `Dir` so the message is a bit more informative.

Before:

```
ERROR: found no collectors for /home/ran/src/pytest/testing/test_compat.py
```

After:

```
ERROR: not found: /home/ran/src/pytest/testing/test_compat.py
(no match in any of [<Dir testing>])
```

The relevant code in `Session.perform_collect` is:

```py
            if self._notfound:
                errors = []
                for arg, collectors in self._notfound:
                    if collectors:
                        errors.append(
                            f"not found: {arg}\n(no match in any of {collectors!r})"
                        )
                    else:
                        errors.append(f"found no collectors for {arg}")

                raise UsageError(*errors)
```

Reply:
Thanks for the clarification